### PR TITLE
Add initial readme, with developer instructions

### PR DESCRIPTION
**Pull request by jonc125**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# The 'Munroe Meter' Jargon Scoring System
+
+## Getting started as a developer
+
+It is best practice to work on Python projects within a virtual environment,
+to avoid conflicts with your main system installation. The `virtualenv` tool
+can be installed following the instructions at
+https://virtualenv.pypa.io/en/stable/installation/
+
+Clone this git repository, then navigate to the folder where you cloned it
```
Comment (rgaiacs) [inline readme.md]: "git" -> "Git"

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+in a terminal and run the following sequence of commands to set up a virtual
+environment and install all the project's dependencies.
+
+```
+virtualenv -p python3 venv
+source venv/bin/activate
+pip install -r requirements.txt
```
Comment (rgaiacs) [inline readme.md]: I think this isn't going o work now.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+```
+pytest
+```
+
+The `requirements.txt` file used above is generated from a specification in
+`requirements.in` by [pip-tools][pip-tools]. This ensures that we list the
+exact versions used of all our dependencies, including indirect ones. If you
+are adding a new dependency, add it to `requirements.in` and then run
+```
```
Comment (rgaiacs) [inline readme.md]: New line here.

File: readme.md (hunk 2 above)
Comment (jonc125) [inline readme.md]: I just ran it locally and it worked.

**Pull request merged**